CodeChat/CommentDelimiterInfo.py

```diff
@@ -155,14 +136,39 @@
   ##                 --
   'Ada':            ( 2,   None,         None),
   'Eiffel':         ( 2,   None,         None),
-  'Vhdl':           ( 2,   None,         None),
-  # ``*>`` as an inline comment is not supported.
-  ##                  * or /
-  'COBOL':          ( 1,   None,         None),
   ##                  ;
   'INI':            ( 1,   None,         None),
   ##                  #
   'YAML':           ( 1,   None,         None),
+
+  # These languages have failing unit tests
+  # ---------------------------------------
+  ##                 //,     /*,            */
+  'Sass':           ( 2,      2,            2),
+  # Or ``#`[`` ~ ``]``, or any other pair.
+  ##                  #,    #'(,             )
+  'Perl6':          ( 1,      3,            1),
+  # ``*>`` as an inline comment is not supported.
+  ##                  * or /
+  'COBOL':          ( 1,   None,         None),
+
+  # ``/*`` ~ ``*/`` tests fail.
+  ##                  #,     /*,            */
+  'GAS':            ( 1,      2,            2),
```

Comment:
Go ahead and move this to the "passing" list, and also note in this file the lack of /* */ command support.

Also, Perl6 tests 48_a and 48_b now pass with my fix -- would you uncomment them?